Lib/transform.py

```diff
@@ -0,0 +1,453 @@
+#!/usr/bin/env python3
+
+"""
+Module with common geodetic transformations.
+
+Product of: 
+U.S. Nat.'l Geodetic Survey / Nat.'l Oceanic Atmospheric Admin.
+1315 East-West Hwy.
+Sivler Spring, MD, 20910
+
+Authors:
+Sungpil Yoon
+Phillip McFarland
+
+Contact:
+phillip.mcfarland@noaa.gov
+
+As required by 17 U.S.C. § 403, third parties producing copyrighted works consisting predominantly 
+of the material appearing in NGS Web pages must provide notice with such work(s) identifying the NGS 
+material incorporated and stating that such material is not subject to copyright protection.
+"""
+
+
+import math as M
+import numpy as np
+import pytest
+
+
+DEG_TO_RAD = M.pi/180
+D2R = DEG_TO_RAD
+R2D = 1/D2R
+
+# define Earth ellipsoid parameters for WGS84:
+
+a = 6378137.0                   # length of semi-major axis (m)
+inv_f = 298.257223563           # reciprical flattening (unitless)
```

Comment:
Why aren't these constants?

Reply:
I don't know what a constant is in the context of Python.

Reply:
See my comment above about constants. You actually define three constants right before this. Values that cannot be changed.

Reply:
Python doesn't have "constants" in the way that c++ or fortran do. In your previous comment about constants I understand the value of not re-using the same string over and over again, instead assigning the string to a variable and using the variable. But in this case, I understand your question to be, why aren't these variables named with all caps. Is that correct?